tests/test_posixDatastoreFits.py

```diff
@@ -0,0 +1,178 @@
+#
+# LSST Data Management System
+#
+# Copyright 2008-2018  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import os
+import unittest
+
+import lsst.utils.tests
+
+from lsst.daf.butler.datastores.posixDatastore import PosixDatastore, DatastoreConfig
+from datasetsHelper import FitsCatalogDatasetsHelper
+
+try:
+    import lsst.afw.table
+    import lsst.afw.image
+    import lsst.afw.geom
+except ImportError:
+    lsst.afw.table = None
+    lsst.afw.image = None
+
+
+class PosixDatastoreFitsTestCase(lsst.utils.tests.TestCase, FitsCatalogDatasetsHelper):
+
+    @classmethod
+    def setUpClass(cls):
+        if lsst.afw.table is None:
+            raise unittest.SkipTest("afw not available.")
+
+    def setUp(self):
+        self.testDir = os.path.dirname(__file__)
+        self.configFile = os.path.join(self.testDir, "config/basic/butler.yaml")
+
+    def testConstructor(self):
+        datastore = PosixDatastore(config=self.configFile)
+        self.assertIsNotNone(datastore)
```

Comment:
We should probably test also that the factory function returns the proper `PosixDatastore` instance.

Reply:
This is your original test code so I haven't looked at it at all.